src/OrchardCore.Modules/OrchardCore.Contents/Views/Admin/List.cshtml

```diff
@@ -82,10 +82,7 @@
                             @T["Actions"]
                         </button>
                         <span class="dropdown-menu dropdown-menu-right" aria-labelledby="bulk-action-menu-button">
-                            @foreach (var item in Model.Options.ContentsBulkAction)
-                            {
-                                <a class="dropdown-item" href="javascript:void(0)" data-action="@item.Value" data-title="@T["Bulk Action"]" data-message="@T["Are you sure you want to {0} these items?", @item.Text.ToLower()]">@item.Text</a>
-                            }
+                            @await DisplayAsync(await New.Contents__BulkActions(BulkActions: Model.Options.ContentsBulkAction))
```

Comment:
Looking for reviews on this. The idea is to use a shape, and then `IShapeTableProviders` to extend the Contents Admin List so features such as these, can provide `BulkActions` via an `IShapeTableProvider` (see comment above).

If this is a `good` technique for extensibility, I would on another pr, a `Shape` that can be used to extend the content list filters (to filter by localization etc), noting that we already have query filter providers, that can mutate the content items query, we just need a `Shape` to display the extendable options. (refer `IContentAdminFilters`)